AmplifyPlugins/Predictions/AWSPredictionsPlugin/Service/CoreML/CoreMLPredictionService.swift

```diff
@@ -13,7 +13,7 @@ class CoreMLPredictionService: CoreMLPredictionBehavior {
 
     let coreMLPlugin: CoreMLPredictionsPlugin
 
-    init(configuration: Any) throws {
+    init(configuration: Any?) throws {
```

Comment:
curious about this change, was it a SwiftLint or a Swift compiler warning?

Reply:
It was Swift Compiler Warning from the calling site. 
"Expression implicitly coerced from Any? to Any"